compiler/scripts/extract-and-check

```diff
@@ -1,25 +1,27 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

CI runners have a single entry in `/bin/`: a `sh` symlink.

Reply:
Thanks !